pkg/tenant/members_test.go

```diff
@@ -227,9 +275,24 @@ func (suite *tenantTestSuite) TestMemberList() {
 	claims.Permissions = []string{tenant.ReadMemberPermission}
 	require.NoError(suite.SetClientCredentials(claims), "could not set client credentials")
 
-	// TODO: Test length of values assigned to *api.MemberPage
-	_, err = suite.client.MemberList(ctx, &api.PageQuery{})
+	rep, err := suite.client.MemberList(ctx, &api.PageQuery{})
 	require.NoError(err, "could not list members")
+	require.Len(rep.Members, 3, "expected 3 members")
+
+	// Verify first member data has been populated.
+	require.Equal(members[0].ID.String(), rep.Members[0].ID, "member id should match")
+	require.Equal(members[0].Name, rep.Members[0].Name, "member name should match")
+	require.Equal(members[0].Role, rep.Members[0].Role, "member role should match")
+
+	// Verify second member data has been populated.
+	require.Equal(members[1].ID.String(), rep.Members[1].ID, "member id should match")
+	require.Equal(members[1].Name, rep.Members[1].Name, "member name should match")
+	require.Equal(members[1].Role, rep.Members[1].Role, "member role should match")
+
+	// Verify third member data has been populated.
+	require.Equal(members[2].ID.String(), rep.Members[2].ID, "member id should match")
+	require.Equal(members[2].Name, rep.Members[2].Name, "member name should match")
+	require.Equal(members[2].Role, rep.Members[2].Role, "member role should match")
```

Comment:
This is a great test. One thing you can also do to reduce the repeated code here is to just make it a loop, e.g.

```golang
for i, _ := range members {
    require.Equal(members[i].ID.String(), rep.Members[i].ID, "member id should match")
    ...
}
```

This improves the maintainability a little bit so that if for example we add more fields to the members response in the future we only have to add one line of code instead of three.

Just something to think about but not a necessary change for merging this PR.